test/binding.cc

```diff
@@ -31,6 +31,7 @@ Object InitDate(Env env);
 Object InitDataView(Env env);
 Object InitDataViewReadWrite(Env env);
 Object InitEnvCleanup(Env env);
+Object InitErrorHandlingPrim(Env env);
```

Comment:
Are these names consistent with the file names being used? This will help @deepakrkris with the unit test filtering.

Reply:
@deepakrkris can you double-check this please?